superdesk/templates/translation_changed_subject.txt

```diff
@@ -0,0 +1 @@
+The translated item {{title}} has been changed
```

Comment:
imo we shouldn't use `title` here, either `headline` or just pass the item, to be consistent with other templates and UI

Reply:
I replaced title to headline. At list in this file there is no case to send the item to the template.